src/app/series/directives/series-templates.component.ts

```diff
@@ -51,8 +54,8 @@ import {
                 to validate the specific file.
               </div>
               <publish-file-template *ngFor="let t of v.fileTemplates" [file]="t" [version]="v"></publish-file-template>
-              <button tabindex=-1 class="add-segment" *ngIf="canAddFile(v)"
-                (click)="addFile(v)"><i class="icon-plus"></i>Add Segment</button>
+              <button tabindex=-1 class="add-segment" *ngIf="canAddFile(v)" type="button"
```

Comment:
This looks like the same bug as the one with the delete modal, so it may not get fixed in this PR.